metrics/mixin/build/example.jsonnet

```diff
@@ -0,0 +1,7 @@
+local kp = (import 'jsonnet/kube-prometheus.libsonnet') + {
```

Comment:
can we rename file to `internal.jsonnet`?

Reply:
sure